File: Codigo/Obligatorio/codigo/ArenaGestor/ArenaGestor.APIContracts/Soloist/SoloistResultConcertDto.cs
using System;

namespace ArenaGestor.APIContracts.Soloist
{
    public class SoloistResultConcertDto
    {
        public int ConcertId { get; set; }
        public string TourName { get; set; }
        public DateTime Date { get; set; }
        public int TicketCount { get; set; }
        public double Price { get; set; }
        public SoloistResultLocationDto Location { get; set; }

    }
}

File: Codigo/Obligatorio/codigo/ArenaGestorFront/src/app/models/Soloists/SoloistResultConcertDto.ts
import { SoloistResultLocationDto } from "./SoloistResultLocationDto";

export class SoloistResultConcertDto {
    concertId: Number = 0;
    tourName: String = "";
    date: Date = new Date();
    ticketCount: Number = 0;
    price: Number = 0;
    location: SoloistResultLocationDto = new SoloistResultLocationDto();
}